syncer/ddl.go

```diff
@@ -86,12 +86,12 @@ func (s *Syncer) parseDDLSQL(sql string, p *parser.Parser, schema string) (resul
 		}, nil
 	case ast.DMLNode:
 		// if DML can be ignored, we do not report an error
-		schema2, table, err2 := tableNameForDML(node)
+		table, err2 := tableNameForDML(node)
 		if err2 == nil {
-			if len(schema2) > 0 {
-				schema = schema2
+			if len(table.Schema) == 0 {
+				table.Schema = schema
```

Comment:
this change seems inconsistent with old logic?

Reply:
I don't think so. Please check it again.

Reply:
old: use schema from ast node 
new: always use the schema passed in 🤔

Reply:
only use `schema` in 

```go
if len(table.Schema) == 0 {
```